src/renderer_d3d11.cpp

```diff
@@ -3425,6 +3429,52 @@ namespace bgfx { namespace d3d11
 			}
 		}
 
+#if BX_PLATFORM_WINRT
```

Comment:
Again, if this compiles on Windows, you should just remove ifdefery...

Reply:
Fixed.